### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by egashhhh**
- [DEMO LINK](https://egashhhh.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://egashhhh.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,8 +11,54 @@
...
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="i-mac"
+        class="card__photo"
+      />
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```
Comment (anastasiiavorobiova) [inline src/index.html:39]:
```suggestion
      <h2 class="card__name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```
Consider leveraging semantic tags for titles

File: src/index.html [hunk 2]
```diff
@@ -11,8 +11,54 @@
...
+      <div class="card__code">Product code: 195434</div>
```
Comment (anastasiiavorobiova) [inline src/index.html:40]:
```suggestion
      <p class="card__code">Product code: 195434</p>
```
Consider leveraging semantic tags for text

File: src/index.html [hunk 3]
```diff
@@ -11,8 +11,54 @@
...
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__review-count">Reviews: 5</div>
```
Comment (anastasiiavorobiova) [inline src/index.html:49]:
```suggestion
        <p class="card__review-count">Reviews: 5</p>
```

File: src/index.html [hunk 4]
```diff
@@ -11,8 +11,54 @@
...
+      </div>
+      <div class="card__price">
+        <div class="card__price-header">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
```
Comment (anastasiiavorobiova) [inline src/index.html:54]:
```suggestion
      <p class="card__price">
        <span class="card__price-header">Price:</span>
        <span class="card__price-value">$2,199</span>
      </p>
```

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,108 @@
...
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:19]:
```suggestion
  padding: 32px 16px 16px;
  
```

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,108 @@
...
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:25]:
```suggestion
  }
  
```

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,108 @@
...
+  &__review-count {
+    text-align: right;
+    color: $main-acent;
+  }
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:72]: Consider using variables for repeating colors

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,108 @@
...
+  flex-direction: row;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    height: 16px;
+    margin-right: 4px;
+    width: 16px;
+    background-repeat: no-repeat;
+  }
+  &.stars--4 :nth-child(-n + 4) {
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:102]:
```suggestion
  &.stars--1 :nth-child(-n + 1),
  ...
  &.stars--4 :nth-child(-n + 4),
  &.stars--5 :nth-child(-n + 5)  {
```
All modifiers should be working

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,3 +1,108 @@
...
+    background-image: $bc-image;
+  }
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:104]:
```suggestion
  }
  
```

File: src/styles/index.scss [hunk 10]
```diff
@@ -1,3 +1,108 @@
...
+    &:hover {
+      background: #fff;
+      color: $btn-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:93]:
```suggestion
  flex-direction: row;
  
```

File: src/styles/index.scss [hunk 11]
```diff
@@ -1,3 +1,108 @@
...
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    height: 16px;
+    margin-right: 4px;
+    width: 16px;
+    background-repeat: no-repeat;
+  }
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:101]:
```suggestion
  }
  
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a few comments to fix

File: src/styles/index.scss (hunk 8 above)
Comment (egashhhh) [reply src/styles/index.scss:102]: не понял что тут сделать

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Feel free to ask for help in the chat to fix comments. 
- Consider using empty new lines between selectors
- All classes with modifiers should be working 
- Consider using variables for repeating colors

File: src/styles/index.scss (hunk 8 above)
Comment (etojeDenys) [reply src/styles/index.scss:122]: you need to reuse all styles from stars task
image

File: src/styles/index.scss [hunk 12]
```diff
@@ -1,3 +1,129 @@
...
+    background: $btn-turquoise-color;
+
+    &:hover {
+      background: #fff;
+      color: $btn-turquoise-color;
+    }
+  }
+}
+
+.stars {
```
Comment (etojeDenys) [inline src/styles/index.scss:109]: image
each BEM block should be in a separate file

File: src/styles/index.scss [hunk 13]
```diff
@@ -1,3 +1,129 @@
...
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    margin-top: 0;
+    height: 16px;
+    width: 166px;
```
Comment (etojeDenys) [inline src/styles/index.scss:77]: do we really need this style? looks redundant for me

**Review by etojeDenys — CHANGES_REQUESTED** [review]
also you should pass all tests, don't hesitate to ask for some help in the chat how to pass them

File: src/styles/index.scss (hunk 13 above)
Comment (egashhhh) [reply src/styles/index.scss:77]: я художник я так вижу))

Comment (egashhhh) [reply src/styles/index.scss:77]: из за того, что мне сказали вместо дива поставить параграфы и спан оно автоматически становится больше, поэтому задал высоту

**Review by etojeDenys — CHANGES_REQUESTED** [review]
good job, but you need to pass your tests, pls, feel free to ask about that in the chat

**Pull request closed without merging**